docs/changelog.md

```diff
@@ -5,6 +5,17 @@ Lines for version numbers should always be formatted as
 with nothing else on the line.
 -->
 * HEAD
+* [0.24.0](https://pypi.org/project/sematic/0.24.0/)
+    * [feature] Introduction of RayCluster[^1]
+    * [feature] "Tee" cloud function logs so they appear in Sematic dashboard and the pod logs
+    * [improvement] Visualize better which run was the true "root failure" when a resolution fails
+    * [improvement] Improve an error message when using untyped dicts in type annotations
+    * [improvement] Various improvements to Sematic internal test infrastructure
+    * [bugfix] Remove a possible server crash for local servers running on Macs
+    * [bugfix] Eliminate a bug that could leave runs hanging if multiple cancellation events were sent
+    * [bugfix] Fix a bug that removed a useful default for image tag in the Helm chart
+    * [bugfix] Rename an incorrectly named helm-chart value for Slack integration
+    * [bugfix] Gracefully terminate runs when a resolver pod restarts mid-resolution
```

Comment:
My philosophy with release notes is that they should only cover things that have a direct user impact. I'm happy to add these things, because I don't think it hurts much (aside from being a bit cluttering) but I don't think they're really relevant to the target audience. Re: documentation, that one might be relevant to folks if they have already checked out the docs and want to know if they should go back, but wouldn't necessarily be relevant to somebody deciding which version of our app to use/upgrade to (since the hosted docs are always the latest on `main`). Thoughts?

Reply:
As a user, I am distraught when I see what I believe to be a regression in a library / framework that I use, with the release notes not mentioning anything, but seeing the code actually having changed.

I believe mentioning everything is the transparent way to go.

Also, another PR has just been merged: #549.

Reply:
discussed offline; I think I'll leave it as-is